bundle/vim-matchup/README.md

```diff
@@ -713,7 +713,7 @@ surround replacement rules but is not currently supported.
 
 ### Auto-closing plugins
 
-match-up does not provide auto-complete or auto-insertion of matches.
+match-up does not provides auto-complete or auto-insertion of matches.
```

Comment:
It should be `does not provide`.

Reply:
I should pay more attention to where i'm `sed`ing from now on :/